data/tabular/SIDER/meta.yaml

```diff
@@ -0,0 +1,223 @@
+      names:
+          - noun: vascular disorders
+          - noun: vascular system disorders
+    - id: blood_and_lymphatic_system_disorders
+      description: blood and lymphatic system disorders
+      type: boolean
+      names:
+          - noun: blood and lymphatic system disorders
+          - noun: disorders of the blood and lymphatic system
+    - id: skin_and_subcutaneous_tissue_disorders
+      description: skin and subcutaneous tissue disorders
+      type: boolean
+      names:
+          - noun: skin and subcutaneous tissue disorders
+          - noun: disorders of the skin and subcutaneous tissue
+    - id: congenital_familial_and_genetic_disorders
+      description: congenital, familial and genetic disorders
+      type: boolean
+      names:
+          - noun: congenital, familial and genetic disorders
+          - noun: familial, congenital and genetic disorders
+    - id: infections_and_infestations
+      description: infections and infestations
+      type: boolean
+      names:
+          - noun: infections and infestations
+          - noun: infestations and infections
+    - id: respiratory_thoracic_and_mediastinal_disorders
+      description: respiratory, thoracic and mediastinal disorders
+      type: boolean
+      names:
+          - noun: respiratory, thoracic and mediastinal disorders
+          - noun: respiratory and thoracic disorders
+    - id: psychiatric_disorders
+      description: psychiatric disorders
+      type: boolean
+      names:
+          - noun: psychiatric disorders
+          - noun: mental health and psychiatric disorders
+    - id: renal_and_urinary_disorders
+      description: renal and urinary disorders
+      type: boolean
+      names:
+          - noun: renal and urinary disorders
+          - noun: kidney and urinary tract disorders
+    - id: pregnancy_puerperium_and_perinatal_conditions
+      description: pregnancy, puerperium and perinatal conditions
+      type: boolean
+      names:
+          - noun: pregnancy, puerperium and perinatal conditions
+          - noun: pregnancy, childbirth, and newborn conditions
+    - id: ear_and_labyrinth_disorders
+      description: ear and labyrinth disorders
+      type: boolean
+      names:
+          - noun: ear and labyrinth disorders
+          - noun: ear and inner ear disorders
+    - id: cardiac_disorders
+      description: cardiac disorders
+      type: boolean
+      names:
+          - noun: cardiac disorders
+          - noun: cardiovascular disorders
+    - id: nervous_system_disorders
+      description: nervous system disorders
+      type: boolean
+      names:
+          - noun: nervous system disorders
+          - noun: disorders of the nervous system
+    - id: injury_poisoning_and_procedural_complications
+      description: injury, poisoning and procedural complications
+      type: boolean
+      names:
+          - noun: injury, poisoning and procedural complications
+          - noun: injuries and poisonings
```

Comment:
i'm still not sure about this. You put in SMILES and then predict something about "procedural complications" which sounds more like surgery

Reply:
@AdrianM0, what are your thoughts on this one?